Apps/AdminWebClient/src/ClientApp/src/views/Support.vue

```diff
@@ -87,6 +89,17 @@ export default class SupportView extends Vue {
         return phn !== null ? phn : "-";
     }
 
+    private get phnSelected(): boolean {
+        return (
+            this.selectedQueryType !== null &&
+            this.selectedQueryType === QueryType.PHN
+        );
```

Comment:
Nit: this condition can be simplified to `this.selectedQueryType === QueryType.PHN`.

Reply:
Changed.